leakcanary-android-leak-deobfuscation-plugin/gradle.properties

```diff
@@ -0,0 +1,3 @@
+POM_ARTIFACT_ID=leakcanary-android-leak-deobfuscation-plugin
+POM_NAME=LeakCanary Android Leak Deobfuscation Plugin
+POM_PACKAGING=jar
```

Comment:
POM_DESCRIPTION ?

Reply:
Other modules don't specify the description so I intentionally haven't added this to make it consistent with the rest of the project.